jupyterhub_moss/batch_script.sh

```diff
@@ -16,5 +16,5 @@
 set -euo pipefail
 
 trap 'echo SIGTERM received' TERM
-
+{{prologue}}
```

Comment:
I used `prologue` here rather than a more specific implementation to keep close to batchspawner which uses the same.